src/components/LoanCards/Buttons/LendIncrementButton.vue

```diff
@@ -123,6 +132,8 @@ export default {
 
 	.lend-increment-button {
 		margin-bottom: 0;
+		flex-grow: 1;
+		margin-left: 0.8rem;
```

Comment:
This expands the width of our button to fix empty space on small screens and for the responsive loan card.